htdocs/libraries/icms/form/Element.php

```diff
@@ -116,6 +116,13 @@ abstract class icms_form_Element {
 	 * @var	string
 	 */
 	private $_description = "";
+
+	/**
+	 * template for this field
+	 */
+	protected icms_view_Tpl $_tpl;
```

Comment:
can we for new vars not use `_` prefix? it's doesn't make sense

Reply:
I believe it was in the early PHP4 days a convention to show that it was a protected variable. You are right, it doesn't make any sense anymore and I'll change it.